packages/sn-list-controls-react/src/ContentList/ContentList.tsx

```diff
@@ -104,7 +104,7 @@ export interface ContentListProps<T extends GenericContent = GenericContent> {
   /**
    * Optional custom selection component
    */
-  getSelectionControl?: (selected: boolean, content: T) => JSX.Element
+  getSelectionControl?: (selected: boolean, content: T, callBack?: Function) => JSX.Element
```

Comment:
I would not make this optional. This callback is needed for the custom component to work properly. Also the name should be onChange or onChangeCallback and its type is `() => void`